src/hooks/useMutation.ts

```diff
@@ -25,7 +24,8 @@ export const useMutation = <T = any, V = object>(
   query: DocumentNode | string
 ): UseMutationResponse<T, V> => {
   const client = useClient();
-  const [state, setState] = useImmediateState<UseMutationState<T>>({
+
+  const [state, setState] = useState<UseMutationState<T>>({
```

Comment:
Small note on this change; This is not a breaking change. Since mutations are realistically run only after a component has mounted, `useImmediateState` was never doing anything here